aui.views/src/AUI/Platform/ABaseWindow.cpp

```diff
@@ -532,3 +546,28 @@ bool ABaseWindow::onGesture(const glm::ivec2& origin, const AGestureEvent& event
     }
     return v;
 }
+
+void ABaseWindow::processTouchscreenKeyboardRequest() {
+    switch (mKeyboardRequestedState) {
+        case ATouchscreenKeyboardState::SHOWN:
+            if (mKeyboardState != ATouchscreenKeyboardState::SHOWN) {
+                requestTouchscreenKeyboardImpl();
+                mKeyboardState = ATouchscreenKeyboardState::SHOWN;
+                emit touchscreenKeyboardShow;
```

Comment:
Verbs in signals should be in the 3rd form. `touchscreenKeyboardShown` `touchscreenKeyboardHidden`